src/EditorFeatures/CSharpTest/AutomaticCompletion/AutomaticLineEnderTests.cs

```diff
@@ -1258,6 +1258,74 @@ public class Bar
 }");
         }
 
+        [WpfFact]
+        public void TestNonEmptyGetAccessor()
+        {
+            Test(@"
+public Class Bar
+{
+    public int P
+    {
+        get
+        {
+            if (true)
+            $$
+            {
+                return 1;
+            }
+        }
+    }   
+}",
+                @"
+public Class Bar
+{
+    public int P
+    {
+        get
+        {
+            i$$f ($$true$$)$$
+            {
+                return 1;
+            }
+        }
+    }   
+}");
```

Comment:
can you explain how i should read these tests?  i'm not really getting what they're trying to indicate.

Reply:
The first string passed to `Test` is the _resulting_ string you get if you invoke Smart Break Line at any of the positions marked with `$$` in the second string.

Reply:
:) Sam has answered the question

Reply:
Ick. That's super confusing :-)